pkg/services/publicdashboards/database/database.go

```diff
@@ -267,3 +268,20 @@ func (d *PublicDashboardStoreImpl) Delete(ctx context.Context, orgId int64, uid
 
 	return affectedRows, err
 }
+
+func (d *PublicDashboardStoreImpl) FindByDashboardFolder(ctx context.Context, dashboard *dashboards.Dashboard) ([]*PublicDashboard, error) {
+	if dashboard == nil || !dashboard.IsFolder {
+		return nil, nil
+	}
+
+	var pubdashes []*PublicDashboard
+
+	err := d.sqlStore.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		return sess.SQL("SELECT * from dashboard_public WHERE dashboard_uid IN (SELECT uid FROM dashboard WHERE folder_id = ? AND org_id = ?)", dashboard.ID, dashboard.OrgID).Find(&pubdashes)
```

Comment:
```suggestion
		return sess.SQL("SELECT * from dashboard_public WHERE (dashboard_uid, org_id) IN (SELECT uid, org_id FROM dashboard WHERE folder_id = ?)", dashboard.ID).Find(&pubdashes)
```

Reply:
whats the difference here? The dashboard uids in the subquery are already scoped by orgId

Reply:
Because the `uid` uniqueness in a dashboard is only valid inside an `orgId`. 
The same `uid` could live in two different orgs, so when you get an `uid` by folder then you will get the public dashboards for the same `uid` but all orgs. 
I know, this is VERY unlikely to happen but I'm pointing this out just to be consistent with how we query the model.

Reply:
fixed 👍